app/javascript/App/bundles/CoinShow/FundamentalsList.tsx

```diff
@@ -54,58 +57,66 @@ class FundamentalsList extends React.Component<Props, {}> {
     const hasIcoEnded = ico_end_epoch < moment().unix()
 
     return (
-      <Grid container={true} direction="column">
-        {hasStartDate && (
-          <>
-            <Grid item={true} className={classes.title}>
-              Start date
+      <CurrencyContext.Consumer>
+        {({ currency, currencyRate }: CurrencyContextType) => {
```

Comment:
Is this `currency` and `currencyRate` used in this component?

Reply:
It was previously, but they took out all the fundamentals that use it. I was going to rip it out but there's still on-going discussion of adding some price-related values back in so I would prefer to leave it even if it's useless right now.